code/datums/trading/food.dm

```diff
@@ -2,15 +2,24 @@
 	name = "Pizza Shop Employee"
 	name_language = TRADER_DEFAULT_NAME
 	origin = "Pizzeria"
-	possible_origins = list("Papa Joe's", "Pizza Ship", "Dominator Pizza", "Little Kaezars", "Pizza Planet", "Cheese Louise")
-	trade_flags = TRADER_MONEY
-	possible_wanted_items = list() //They are a pizza shop, not a bargainer.
-	possible_trading_items = list(/obj/item/weapon/reagent_containers/food/snacks/sliceable/pizza   = TRADER_SUBTYPES_ONLY)
+	possible_origins = list("Papa Joe's", "Mamma Mia", "Dominator Pizza", "Little Kaezars", "Pizza Planet", "Cheese Louise")
+	trade_flags = TRADER_GOODS|TRADER_MONEY|TRADER_WANTED_ONLY
+
+	possible_wanted_items = list(
+		/obj/item/weapon/material/kitchen/utensil 			= TRADER_SUBTYPES_ONLY, // Customers keep stealing their cutlery
+		/obj/item/weapon/material/kitchen/utensil/fork/fluff = TRADER_BLACKLIST_ALL, // No custom items
+		/obj/item/weapon/material/kitchen/utensil/knife/boot = TRADER_BLACKLIST // No boot knifes
+		)
+
+	possible_trading_items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/sliceable/pizza   = TRADER_SUBTYPES_ONLY
+		)
 
 	speech = list(
 		"hail_generic"    = "Hello! Welcome to ORIGIN, may I take your order?",
 		"hail_deny"         = "Beeeep... I'm sorry, your connection has been severed.",
 		"trade_complete"    = "Thank you for choosing ORIGIN!",
+		"what_want"         = "Customers keep stealing these! We need new",
```

Comment:
```suggestion
		"what_want"         = "Customers keep stealing these! We need new.",
```

Reply:
- unresolved - just now looked at the code again and thought about it - this conversation option lists a pool of items after it to tell the merchant what the trader wants right now. 
" It's very odd, but I need. kitchen knife, plastic fork." is very derp.

Re-committing previous version and resolving this and it's sister thread with same. [Code is like this on all traders that buy items.]